src/Adapters/EventSource.Netstandard13/EventSourceNamingMatchRuleBase.cs

```diff
@@ -0,0 +1,49 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="EventSourceNamingMatchRuleBase.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.ApplicationInsights.EventSourceListener
+{
+    /// <summary>
+    /// Abstraction of a match rule for event source. To be inherited by either an enabling rule or a disabling rule, etc.
+    /// </summary>
+    public abstract class EventSourceNamingMatchRuleBase
```

Comment:
Since there is is no "EventSourceNamingMatchRule" of any kind, let's name this EventSourceListeningRequestBase

Reply:
Sure. Will be fixed in the next iteration.